test/workbox-range-requests/integration/range-requests-plugin.js

```diff
@@ -0,0 +1,76 @@
+const expect = require('chai').expect;
+const seleniumAssistant = require('selenium-assistant');
+
+describe(`rangeRequests.Plugin`, function() {
+  let webdriver;
+  let testServerAddress = global.__workbox.server.getAddress();
+
+  beforeEach(async function() {
+    if (webdriver) {
+      await seleniumAssistant.killWebDriver(webdriver);
+      webdriver = null;
+    }
+
+    global.__workbox.server.reset();
+
+    // Allow async functions 10s to complete
+    webdriver = await global.__workbox.seleniumBrowser.getSeleniumDriver();
+    webdriver.manage().timeouts().setScriptTimeout(10 * 1000);
+  });
+
+  after(async function() {
+    if (webdriver) {
+      await seleniumAssistant.killWebDriver(webdriver);
+    }
+  });
+
+  const activateSW = async (swFile) => {
+    const error = await webdriver.executeAsyncScript((swFile, cb) => {
+      navigator.serviceWorker.register(swFile).then(() => {
+        navigator.serviceWorker.addEventListener('controllerchange', () => {
+          if (navigator.serviceWorker.controller.scriptURL.endsWith(swFile)) {
+            cb();
+          }
+        });
+      }).catch((error) => cb(error.message));
+    }, swFile);
+
+    if (error) {
+      throw error;
+    }
+  };
+
+  it(`should return a partial response that satisfies the request's Range: header, and an error response when it can't be satisfied`, async function() {
+    const testPageUrl = `${testServerAddress}/test/workbox-range-requests/static/`;
+    const swUrl = `${testPageUrl}sw.js`;
+    const dummyUrl = `${testPageUrl}dummy.txt`;
```

Comment:
Stupid thing, but could you change `dummy.txt` to `this-file-doesnt-exist.txt` just to make it painfully clear that it HAS to come from the cache and via the SW.